sphinx_es_suggest/extension.py

```diff
@@ -1,25 +1,33 @@
 import os
 from sphinx.util.fileutil import copy_asset
 
-ASSETS_FILES = [
-    os.path.join('js', 'sphinx_es_suggest.js'),
-    os.path.join('css', 'sphinx_es_suggest.css'),
+CUSTOM_ASSETS_FILES = [
+    os.path.join('js', 'sphinx_es_suggest.min.js'),
+    os.path.join('css', 'sphinx_es_suggest.min.css'),
 ]
 
 
 def copy_asset_files(app, exception):
     if exception is None:  # build succeeded
-        for f in ASSETS_FILES:
+        for f in CUSTOM_ASSETS_FILES:
             path = os.path.join(os.path.dirname(__file__), '_static', f)
             copy_asset(path, os.path.join(app.outdir, '_static', f.split('.')[-1]))
 
 
 def setup(app):
 
     app.connect('build-finished', copy_asset_files)
+    
+    # promise-polyfill to support older browsers (~3.1kB).
+    # https://github.com/taylorhakes/promise-polyfill
+    app.add_js_file('//cdn.jsdelivr.net/npm/promise-polyfill@8/dist/polyfill.min.js')
```

Comment:
Is it possible to package the polyfills with the extension? I would rather we do that so that the generated documentation is self contained and doesn't rely on an internet connection.

Reply:
I have removed the polyfills. [#3 (comment)](https://github.com/rtfd/readthedocs-sphinx-search/pull/3#issuecomment-500392463).